content/04.study.md

```diff
@@ -47,6 +47,25 @@ Deep learning applied to gene expression data is still in its infancy, but the f
 Many previously untestable hypotheses can now be interrogated as deep learning enables analysis of increasing amounts of data generated by new technologies.
 For example, the effects of cellular heterogeneity on basic biology and disease etiology can now be explored by single-cell RNA-seq and high-throughput fluorescence-based imaging, techniques we discuss below that will benefit immensely from deep learning approaches.
 
+### DNA Methylation
+
+#### Introduction 
+
+DNA Methylation (DNAm), is the process of adding a methyl group to a cytosine in the context of a CpG dinucleotide. 
```

Comment:
```suggestion
DNA methylation, is the process of adding a methyl group to a cytosine in the context of a CpG dinucleotide. 
```
Didn't look like DNAm was ever used. I'd remove it unless it's going to be used elsewhere.

Reply:
agree